docs/source/overview.rst

```diff
@@ -0,0 +1,249 @@
+.. image:: https://badges.gitter.im/cloud-custodian/cloud-custodian.svg
```

Comment:
this is the readme, it shouldn't be added to the repo again. there's some other work in progress to make that more clear and prevent this in the future.